### DISCUSSION TITLE
getting 403 on fresh intall on production

### DISCUSSION THREAD

**Discussion by abhijeetpandeyOO7** [filamentphp/filament#14312] [Help]
### Package

Other

### Package Version

v3.2

### How can we help you?

hey i installed fresh copy of laravel 11.9 and filament 3,2 

but getting 403 error on login 
![image](https://github.com/user-attachments/assets/7331a0f4-b396-4536-a34d-fa7b54a8177d)

then I look documentation and applied following to user model 

`<?php

namespace App\Models;

// use Illuminate\Contracts\Auth\MustVerifyEmail;
use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Foundation\Auth\User as Authenticatable;
use Illuminate\Notifications\Notifiable;
use Filament\Models\Contracts\FilamentUser;
use Filament\Panel;
use Illuminate\Foundation\Auth\User as Authenticatable;
 

class User extends Authenticatable implements FilamentUser
{
    use HasFactory, Notifiable;

    /**
     * The attributes that are mass assignable.
     *
     * @var array<int, string>
     */
    protected $fillable = [
        'name',
        'email',
        'password',
    ];

    /**
     * The attributes that should be hidden for serialization.
     *
     * @var array<int, string>
     */
    protected $hidden = [
        'password',
        'remember_token',
    ];

    /**
     * Get the attributes that should be cast.
     *
     * @return array<string, string>
     */
    protected function casts(): array
    {
        return [
            'email_verified_at' => 'datetime',
            'password' => 'hashed',
        ];
    }
    public function canAccessPanel(Panel $panel): bool
    {
        return $this->is_admin == 1;
    }
    

}`

now I am getting 500 error
![image](https://github.com/user-attachments/assets/04fa9feb-daa0-47f2-95d6-2df6a477e682)

pls help

**Comment by leandrocfe** [comment]
what about the laravel.log in the storage folder?

**Comment by e2d2-dev** [comment]
take a look on browser devtools network tab and or switch on debug in .env
